includes/transformer/class-transformers-manager.php

```diff
@@ -0,0 +1,292 @@
+<?php
```

Comment:
I would remove the `transformers` in the name, because this is already defined by the namespace and I would go with `factory` instead of `manager` because this is a known pattern.

Reply:
Thanks for those suggestions. Another naming I thought about is using `registry`, that is the naming the Gutenberg project uses for its blocks.